cmd/zoekt-sourcegraph-indexserver/sg.go

```diff
@@ -15,15 +15,21 @@ import (
 	"os/exec"
 	"path"
 	"path/filepath"
+	"strconv"
 
 	"github.com/google/zoekt"
 	retryablehttp "github.com/hashicorp/go-retryablehttp"
 )
 
 type Sourcegraph interface {
-	GetIndexOptions(repos ...string) ([]indexOptionsItem, error)
+	GetIndexOptions(repos ...uint32) ([]indexOptionsItem, error)
 	GetCloneURL(name string) string
-	ListRepos(ctx context.Context, indexed []string) ([]string, error)
+	ListRepos(ctx context.Context, indexed []uint32) ([]uint32, error)
```

Comment:
Two comments:
1. I think `ListRepoIDs` is better / more explicit, specially without a custom ID type to hint at that.
2. Can we use `*roaring.Bitmap` instead of uint32 slices?

```suggestion
	ListRepoIDs(ctx context.Context, indexed *roaring.Bitmap) (*roaring.Bitmap, error)
```

Reply:
will update names. roaring bitmap doesn't really help here, since the list of IDs we generate ourselves by iterating over the shards on the filesystem. I'd prefer sticking to simple slices since we don't marshal the bitmaps we marshal lists.

I think this would be a reasonable change to make if we follow-up and add bitmap support to the endpoints in question (something I thought about doing once we have everything functional)